### PR TITLE
Bug: Fix sequencer

### PR DESCRIPTION
**Pull request by magestrio**
## Type

- [ ] Feature
- [X] Bug fix
- [ ] CI/CD
- [ ] Documentation
- [ ] Other

## Description

Brief description of the changes made.

### CODE REVIEW

File: packages/serverless/src/domain/use-case/prover.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
 import ModelDocument from '../../model/abstract/document.js';
 
 import { buildSchema } from './schema.js';
-import { TDocumentField, TMerkleProof } from '@zkdb/common';
+import { ESequencer, TDocumentField, TMerkleProof } from '@zkdb/common';
```
Comment (SangTran-127) [inline packages/serverless/src/domain/use-case/prover.ts:14]: import sorting

File: packages/storage/src/database/common/sequencer.ts [hunk 2]
```diff
@@ -19,26 +21,61 @@ export class ModelSequencer extends ModelBasic<TSequencedItem> {
...
+          updatedAt: creationTime,
+        },
+        { session }
+      );
+
+      if (!insertResult.insertedId) {
+        throw new Error(`Failed to create sequence '${sequenceName}'`);
+      }
+
+      return 1;
```
Comment (SangTran-127) [inline packages/storage/src/database/common/sequencer.ts:67]: dont hardcode

Comment (magestrio) [reply packages/storage/src/database/common/sequencer.ts]: fixed

File: packages/serverless/src/domain/use-case/prover.ts (hunk 1 above)
Comment (magestrio) [reply packages/serverless/src/domain/use-case/prover.ts]: fixed

**Review by chiro-hiro — APPROVED** [review]

**Pull request merged**